src/lib/bcmp/bm/bm_util.h

```diff
@@ -9,8 +9,8 @@ extern "C" {
 #endif
 #include "util.h"
 
-extern const ip6_addr_t multicast_global_addr;
-extern const ip6_addr_t multicast_ll_addr;
+// extern const ip6_addr_t multicast_global_addr;
+// extern const ip6_addr_t multicast_ll_addr;
```

Comment:
now defined in `bm_common/util.h`, which has been updated with the latest bm_core update.